### PR TITLE
Deleted unused file

### PR DESCRIPTION
**Pull request by zzxgzgz**
This PR addresses license mismatch issue for clang format file.
- Upgrade clang version from 7 to 9.
- Correct its license of clang format file to Apache 2.0

### CODE REVIEW

**Review by cj-chung — APPROVED** [review]
LGTM

**Comment by er1cthe0ne** [comment]
Please double check. I believe it is used today, e.g. two space for indent. It is picked up by visual studio code automagically. :)

**Comment by zzxgzgz** [comment]
> Please double check. I believe it is used today, e.g. two space for indent. It is picked up by visual studio code automagically. :)

I just checked. Without this file, my VSCode is still able to format the code the way, I think it should be fine.

File: .clang-format [hunk 1]
```diff
@@ -1,493 +0,0 @@
-# SPDX-License-Identifier: GPL-2.0
-#
-# clang-format configuration file. Intended for clang-format >= 4.
-#
-# For more information, see:
-#
-#   Documentation/process/clang-format.rst
-#   https://clang.llvm.org/docs/ClangFormat.html
-#   https://clang.llvm.org/docs/ClangFormatStyleOptions.html
-#
```
Comment (xieus) [inline .clang-format:10]: Based on https://github.com/angular/clang-format/blob/master/LICENSE, this .clang-format file is supposed to be under Apache 2.0 license. @er1cthe0ne do you still remember why GPL 2.0 is here? copy paste issue or?

Comment (er1cthe0ne) [reply .clang-format:10]: this .clang-format file could possibly be derived from here with its license header: https://github.com/CentaurusInfra/mizar/blob/dev-next/.clang-format

Comment (xieus) [reply .clang-format:10]: Okay, the clang project is under Apache 2.0 after version 9.0. So it should be fine to use Apache 2.0 statement here.

**Pull request merged**